skada/deep/callbacks.py

```diff
@@ -73,3 +74,104 @@ def on_epoch_begin(self, net, dataset_train=None, **kwargs):
             target_kmeans.fit(features_t)
 
         net.criterion__adapt_criterion.target_kmeans = target_kmeans
+
+
+class ComputeMemoryBank(Callback):
```

Comment:
Minor remark: I think `ComputeMemoryBank` and `MemoryBankInit` should be merged. Wdyt?